src/plugins/ResizeReorder/ResizeReorderCell.js

```diff
@@ -0,0 +1,249 @@
+/**
+ * Copyright Schrodinger, LLC
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule ResizeReorderCell
+ * @typechecks
+ */
+
+import FixedDataTableCellDefault from 'FixedDataTableCellDefault';
+import React from 'react';
+import PropTypes from 'prop-types';
+import ResizerKnob from 'ResizerKnob';
```

Comment:
nit: We have a -Knob and a -Handle.  They seem like they same idea, can we pick one or the other and make them the same, in other words, they could be 'ResizeHandle' and 'ReorderHandle'